content/blog/posts/at-the-cross-roads-of-risk-factors-how-the-duncan-unit-became-a-hotbed-for-covid19.md

```diff
@@ -0,0 +1,28 @@
+
+Jeremy Desel, a TDCJ spokesman, stressed that the high positivity rate stemmed from the frequency and large scale effort when it came to testing those incarcerated in Duncan Unit. When asked about the mortality rate in particular, Desel shared: 
+
+“If you look at our offender deaths — and actually, to be completely honest, our staff and offender deaths — if you look at them, across the board, there are very, very few that are not higher-risk folks, in that they have pre-existing conditions and/or are older. So you’re going to see, I think, a higher number of those instances in the older population units like Duncan.”
+
+This past year was not the first time that the vulnerability of older, incarcerated individuals has come to light. [A 2018 TDCJ employee publication](https://tdcj.texas.gov/connections/2018JanFeb/images/JanFeb2018_agency_shelter.pdf) reported that individuals 55 and older access in-prison health care five times more frequently than the rest of the incarcerated population. The article quoted [Dr. Lannette Linthicum](https://www.tdcj.texas.gov/divisions/hsd/index.html), director of TDCJ’s Health Services Division: “If they’re 55, they usually have the physiology of a 65-year-old, due to things such as lack of preventive health care, and behaviors such as alcohol and IV drug abuse.”
+
+However, as people age, they also become far less likely to offend. Developed nearly four decades ago by Travis Hirschi and Michael Gottfredson, the “[age-crime curve](http://troublesofyouth.pbworks.com/f/age+and+the+explanation+of+crime+-+Hirschi+and+Gott.pdf)” has been a guidepost for assessing the likelihood of a returning citizen reoffending. Hirschi and Gottfreson found that, regardless of the person’ sex, criminality spikes between the ages of 12 and 25 after which there is a steep decline until reaching near-zero numbers by the age of 55. According to this model, the overwhelming majority of those incarcerated in the Duncan Unit would pose little to no risk to society upon release.
+
+In fact, more than two-thirds of the Duncan Unit was parole eligible as of July 2020 and most of them (208 of 270) had been parole eligible for more than a year. Of the 21 men who died after contracting COVID while serving time in the Duncan Unit, 13 had served at least half their sentence, 17 were eligible for parole, and all were over the age of 60. Among these men were Jose Faide Morones, 70, who had completed his sentence but was denied parole in May of this year; and Jimmy Ray Price, 82, who’d served 24 years on a 25-year sentence and had been eligible for parole for 11 years, but was denied parole last spring.
```

Comment:
```suggestion
In fact, more than two-thirds of the Duncan Unit was parole-eligible as of July 2020 – and most of those individuals (208 of 270) had been parole-eligible for more than one year. Of the 21 men who died after contracting COVID-19 while serving time at the Duncan Unit, 13 had served at least half their sentence, 17 were eligible for parole, and all 21 were over the age of 60. Among these men were Jose Faide Morones, 70, who had completed his sentence but was denied parole in May 2020; and Jimmy Ray Price, 82, who’d served 24 years on a 25-year sentence and had been eligible for parole for 11 years, but was denied parole last spring.
```

Reply:
would it make sense to add when Morones and Price died? and/or links to their custodial death reports?

Reply:
I'm torn on this. Not sure what adding when they died would add since we are already saying that this is sometime between last March and December throughout the blog so I don't know if saying they died in July or Aug would really add much. I'm intrigued by the idea of linking to the custodial death reports.

Reply:
Great -- the links will give people the date of death if they're curious too. 
```suggestion
In fact, more than two-thirds of the Duncan Unit was parole eligible as of July 2020 and most of them (208 of 270) had been parole eligible for more than a year. Of the 21 men who died after contracting COVID while serving time in the Duncan Unit, 13 had served at least half their sentence, 17 were eligible for parole, and all were over the age of 60. Among these men were [Jose Faide Morones, 70](https://oagtx.force.com/cdr/VIPForm__VIP_FormWizardPDF?id=a2Ct00000038tpEEAQ&templateId=a2x5A000001M2UWQA0), who had completed his sentence but was denied parole in May of this year; and [Jimmy Ray Price, 82](https://oagtx.force.com/cdr/VIPForm__VIP_FormWizardPDF?id=a2Ct0000002lalsEAA&templateId=a2x5A000001M2UWQA0), who’d served 24 years on a 25-year sentence and had been eligible for parole for 11 years, but was denied parole last spring.
```